resources/iam-kubeconfig-service/templates/deployment.yaml

```diff
@@ -43,6 +43,13 @@ spec:
         ports:
           - name: http
             containerPort: 8000
+        readinessProbe:
```

Comment:
You need to provide PR image, your code changes will not work without it, k8s will try to do the probe but the image used in the deployment has older version, without this endpoint.

Reply:
Yep, that's true, forgot about it, thanks!